README.md

```diff
@@ -191,55 +187,57 @@ import { ProviderRpcClient } from '@eversurf/surfkeeper-provider';
 const rpc = new ProviderRpcClient();
```

Comment:
Need to remember that further we are to pass the "net" parameter in the constructor... (it looks like this is suitable for API)